openhtf/plugs/uart.py

```diff
@@ -0,0 +1,295 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
I see. That scheme makes good sense. Let's go with it for now and see how it shapes up.